server/chat.js

```diff
@@ -221,6 +221,7 @@ Chat.namefilter = function (name, user) {
 		if (nameSymbols.length > 4 || /[^a-z0-9][a-z0-9][^a-z0-9]/.test(name.toLowerCase() + ' ') || /[\u00ae\u00a9].*[a-zA-Z0-9]/.test(name)) name = name.replace(/[\u00A1-\u00BF\u00D7\u00F7\u02B9-\u0362\u2012-\u2027\u2030-\u205E\u2050-\u205F\u2190-\u23FA\u2500-\u2BD1\u2E80-\u32FF\u3400-\u9FFF\uF900-\uFAFF\uFE00-\uFE6F]+/g, '').replace(/[^A-Za-z0-9]{2,}/g, ' ').trim();
 	}
 	name = name.replace(/^[^A-Za-z0-9]+/, ""); // remove symbols from start
+	name = name.replace(/@/g, ""); // Remove @ as this is used to indicate status messages
```

Comment:
Don't you want to remove `/@.*$/`  (everything following the `@`, including the `@`?)

Reply:
Oh sorry, I see this is followup to #5440

Reply:
If your name is `f@o@o@b@a@r` I'd rather just strip all the `@`s